plugins/cpp/parser/src/ppincludecallback.cpp

```diff
@@ -77,39 +77,48 @@ void PPIncludeCallback::InclusionDirective(
 
   std::string includedPath = searchPath_.str() + '/' + fileName_.str();
   model::FilePtr included = _ctx.srcMgr.getFile(includedPath);
```

Comment:
Instead of adding extra indentation everywhere, maybe it was simpler to bring the `getFile()` calls from line 79 and 94 to the beginning before line 76 and make an early return if either of them is null. It would indicate better that no action is needed in that case.